### DISCUSSION TITLE
How set number of epochs

### DISCUSSION THREAD

**Discussion by Vichoko** [Lightning-AI/pytorch-lightning#5818] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
How do I set the number of epochs to train?

#### What have you tried?    
Looking for documentation. 
Looking for examples.

**Comment by williamFalcon** [comment]
https://pytorch-lightning.readthedocs.io/en/latest/Trainer/Training%20Loop/#force-training-for-min-or-max-epochs

**Comment by npit** [comment]
Up-to-date link: 
https://pytorch-lightning.readthedocs.io/en/latest/pytorch_lightning.trainer.training_loop.html#force-training-for-min-or-max-epochs

**Comment by ashutoshrdwivedi** [comment]
Up-to-date link : 
https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#max-epochs

**Comment by lyndond** [comment, marked as answer]
Up-to-date link:
https://pytorch-lightning.readthedocs.io/en/latest/common/trainer.html#max-epochs
